cli-plugins/insight/query-somb-reports.hbs.md

```diff
@@ -0,0 +1,140 @@
+# Query Software Bill of Materials Reports
+
+This topic tells you how to query the database to retrieve a Software Bill 
+of Materials (commonly known as SBoM) report for a particular image. The information returned is similar
+to the SBoM returned by [querying for packages and CVE's a specific image contains](query-data.hbs.md#a-idexample1aexample-1-what-packages-and-cves-does-a-specific-image-contain)
+with one major difference: SBoM reports contain Common Vulnerabilities and Exposures(commonly known as CVE) and packages for an image at a specific point in time with a specific vulnerability scan tool.
+Unlike querying for images which return all information linked to an image found in the database. 
+
+## <a id='sup-usecase'></a>Supported use cases
+
+The following are use cases supported by the CLI:
+- What SBoM reports exists for a particular image?
+- Fetching a specific SBoM report.
+
+## <a id='query-insight'></a> Query using the Tanzu Insight CLI plug-in
+
+[Install the Tanzu Insight CLI plug-in](cli-installation.hbs.md) if you have not already done so.
+
+There are two commands for querying SBoM reports.
+
+- `list` - Query for a list of existing SBoM reports associated with an image
+- `get` - Query for a specific SBoM report
+
+
+## <a id='example1'></a>Example #1: What SBoM reports exists for a specific image?
+
+Run:
+
+```console
+tanzu insight report list --digest DIGEST
+```
+
+Where:
+
+- `DIGEST` is the component version or image digest.
```

Comment:
For my information, what does `component version` mean?

Reply:
I copied the same text from the example of getting data from the store using the CLI. I'm assuming this means that the image version could either be a digest or a numbered version. You can see the same text here: https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.7/tap/cli-plugins-insight-query-data.html